packages/react/src/primitives/types/view.ts

```diff
@@ -94,7 +94,8 @@ interface JSXElementForwardRefExoticComponent<P = {}> {
   (props: P): React.ReactElement | null;
   defaultProps?: Partial<P> | undefined;
   displayName?: string | undefined;
-  propTypes?: React.WeakValidationMap<P> | undefined;
+  // deprecated in React@19 but needed for backwards compatibility in previous versions
+  propTypes?: any;
```

Comment:
i think `defaultProps` was removed as well. why is only `propTypes` updated in this case?

Reply:
Discused offline